File: 24 mar 22/multiplication_table.py

class multiplication_table :

    def __init__(self,a):
        self.i=a

    def table(self):
        
        for i in range(1,10):
            ans=self.i*i
            print(ans)



def main():
    x=int(input("Enter number"))
    obj=multiplication_table(x)
    obj.table()
    
if __name__ == "__main__":
    main()

File: 24 mar 22/multiplication_table.js

class multiplication_table
{
	
	constructor(no)
	{
		this.no=no
	}
	
	table()
	{
		var ans;
		for(var i=1;i<=10;i++)
		{
			ans=this.no*i;
			console.log(ans);
		}
		
	}
	
}

const obj1 = new multiplication_table(5);

const obj2 = new multiplication_table(7);

obj1.table();
obj2.table();











